CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - Unreleased
+### Added
+- Added a rake task definition that can be optionally included into a non-Rails project to generate
+  schema migrations.
```

Comment:
Can you put a comment at the top of `lib/declare_schema/rake.rb` saying this is its (only) purpose? And add a bit to the README explaining how to use it with/without Rails?

Reply:
Done 0b6ace6a3ca067c4110987ad58259ed44d933e06